src/site/styles/layout.less

```diff
@@ -109,15 +144,10 @@ input {
     }
 }
 
-#logout .action-x {
+#logout .action-Sign-Out {
```

Comment:
I'm trying not to change too much of the original implementation. Also, the original implementation is very concise because by the time you read this line: 

https://github.com/JoshuaKGoldberg/intern-assassins/blob/master/src/site/scripts/components/profile/actionbutton.tsx#L89

it's easy to know what lines I need to Ctrl+F. If you add a new semantic it's just more churn.

(for the sake of illustrating how it worked before, I'll paste the other original lines involved) 
https://github.com/JoshuaKGoldberg/intern-assassins/blob/master/src/site/styles/layout.less#L112
https://github.com/JoshuaKGoldberg/intern-assassins/blob/master/src/site/scripts/components/apps/appuser.tsx#L72